test/user.test.js

```diff
@@ -0,0 +1,52 @@
+import { get, post, put, del, config } from '../framework/services/apiService'
```

Comment:
В файле `..framework/services/apiService` у вас нет экспорта конфига (и не должно быть)

Чтобы импортировать сюда, конфиг, нужна такая строка:
```suggestion
import { get, post, put, del } from '../framework/services/apiService'
import { config } from '../framework/config/testconfig'
```